cluster-sharding/src/main/scala/com/evolutiongaming/akkaeffect/cluster/sharding/ClusterShardingLocal.scala

```diff
@@ -148,6 +155,14 @@ object ClusterShardingLocal {
                   val shards = allocation().values.flatten.toList
                   context.sender().tell(shards, context.self)
 
+                case Unstash =>
+                  rebalancing = false
+                  stashed.foreach(self.tell(_, context.self))
+                  stashed = Vector.empty
+
+                case msg: ShardRegion.Msg if rebalancing =>
+                  stashed = stashed :+ msg
+
```

Comment:
honestly, I do not get what will cause this stashing and how it will be used

Reply:
here's good explanation how it works https://doc.akka.io/docs/akka/current/typed/cluster-sharding-concepts.html#shard-rebalancing